rocket_sqlite_oracle/migrations/20210809053931_create_tables.sql

```diff
@@ -3,6 +3,7 @@ create table if not exists signatures (
     timestamp       integer not null,
     ask_price       integer not null,
     bid_price       integer not null,
+    units           text    not null,
```

Comment:
Generally, editing migrations isn't really a thing that one should do.

However, in this context, I would say it is okay because this is an example so there is never really a point where we this officially released and we'd have to migrate existing databases. Instead, the migrations are only there to help people get started.